### DISCUSSION TITLE
Donations/sponsorships for OpenOrienteering

### DISCUSSION THREAD

**Discussion by jmacura** [OpenOrienteering/mapper#1996] [General]
Dear @dg0yt and @lpechacek , as the current project leaders,

I wonder, whether you have considered to set up some some donation/sponsoring system for OpenOrienteering or Mapper development per se. Nowadays, there are many popular, easy-to-use platforms, which can handle the process, some of them even integrated to GitHub.

I can see both pros and cons of doing such a thing, so I am really interested in your opinions and why we can't see any "sponsor"/"donate" button anywhere around.

Thanks

**Comment by dg0yt** [comment]
The key issue issue is what "financial entity" to attribute the payments to. Handling it personally is common practice and seems easy, however there are some downsides which are less obvious. I would prefer to use a separate entity. I do have a list of some organizations which offer suitable services including acting as a "financial host" for open source projects. But usually, there was at least one major downside with most of them. Perhaps it is time to revisit that list.
